scalafix-tests/integration/src/main/scala/scalafix/test/ExplicitSynthetic.scala

```diff
@@ -1,10 +1,10 @@
 package scalafix.test
 
-import scalafix.v1.SemanticRule
+import scala.meta._
+
+import scalafix.v1._
 
 class ExplicitSynthetic() extends SemanticRule("ExplicitSynthetic") {
-  import scalafix.v1._
```

Comment:
It seems that in Scala3, the `SemanticRule` reference above is found thanks to this, making the top-level import unused. To get a consistent behavior across Scala versions, I just hoisted them.